src/components/list/list.css

```diff
@@ -0,0 +1,13 @@
+.artsy-list{
+    list-style: none;
```

Comment:
in global css you can add : 

ul {list-style : none ; }

because mostly we want to remove dot form li tag